app/views/fileChecksProgress.scala.html

```diff
@@ -4,26 +4,36 @@
 
 @main(Messages("checkingRecords.header")) {
     @defining(play.core.PlayVersion.current) { version =>
-    <div class="govuk-grid-row">
+    <div class="govuk-grid-row file-check-progress">
         <div class="govuk-grid-column-two-thirds">
 
             @progressIndicator(Messages("checkingRecords.progress"))
             @frontEndInputs(frontEndInfo)
             <h1 class="govuk-heading-l">@Messages("checkingRecords.title")</h1>
-            <p class="govuk-body">@Messages("checkingRecords.description")</p>
             <input id="consignmentId" type="hidden" value="@consignmentId">
-            <div class="file-check-progress">
-                <label for="av-metadata-progress-bar" class="govuk-label">@Messages("checkingRecords.antivirus")</label>
-                <progress id="av-metadata-progress-bar" class="file-check-progress__progress-bar" value="@fileChecks.avMetadataProgressPercentage" max="100"></progress>
-            </div>
-            <div class="file-check-progress">
-                <label for="checksum-progress-bar" class="govuk-label">@Messages("checkingRecords.checksum")</label>
-                <progress id="checksum-progress-bar" class="file-check-progress__progress-bar" value="@fileChecks.checksumProgressPercentage" max="100"></progress>
-            </div>
-            <div class="file-check-progress">
-                <label for="ffid-progress-bar" class="govuk-label">@Messages("checkingRecords.ffid")</label>
-                <progress id="ffid-progress-bar" class="file-check-progress__progress-bar" value="@fileChecks.ffidMetadataProgressPercentage" max="100"></progress>
+            <p class="govuk-body">Please wait while your records are being checked. This may take a few minutes</p>
```

Comment:
Does this need a full stop at the end: `This may take a few minutes.`

Reply:
Yeah, it should have. I've added one in.